client/src/app/site/motions/modules/motion-detail/components/motion-detail-diff/motion-detail-diff.component.html

```diff
@@ -10,12 +10,13 @@
                 [class.amendment]="isAmendment(change)"
                 [class.recommendation]="isChangeRecommendation(change)"
             >
-                <span *ngIf="change.getLineFrom() >= change.getLineTo() - 1" class="line-number">
+                <span *ngIf="change.getLineFrom() >= change.getLineTo() - 1 && !change.isTitleChange()" class="line-number">
```

Comment:
That's a little too much logic for a `*ngIf`. `!change.isTitleChange()` is directly duplicated.
Tunnel through a (commented) function `isLineInRage` (or similar)